addons/website_slides/data/mail_data.xml

```diff
@@ -63,6 +63,24 @@
             <field name="auto_delete" eval="True"/>
         </record>
 
+        <!-- Completed Channel Message -->
+        <record id="completed_channel_template" model="mail.template">
+            <field name="name">Completed Course</field>
+            <field name="model_id" ref="model_slide_channel"/>
+            <field name="subject">Congratulation ! You completed ${object.name}</field>
+            <field name="email_from">${(object.user_id.email_formatted or object.env.ref("base.user_admin").email_formatted) | safe}</field>
```

Comment:
I would use channel.user_id.emial_formatted or user.company_id.catchall_formatted instead of hardcoding odoobot